tox.ini

```diff
@@ -42,7 +42,7 @@ basepython=python
 deps=flake8
 extras=lint
 commands=
-	flake8 {toxinidir}/tests/ethpm {toxinidir}/ethpm 
+	flake8 {toxinidir}/tests/ethpm {toxinidir}/ethpm --exclude ethpm/assets
```

Comment:
post merge cleanup suggestion.  I *think* you can do this in the `pytest.ini` file which will help anyone who manually runs `flake8` to not have to add this to their command.